src/store/annotations/reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { createReducer, combineReducers } from '@reduxjs/toolkit';
+import { AnnotationsState } from './types';
+import { setAnnotation, setAnnotations } from './actions';
+
+const annotationsAllIds = createReducer<AnnotationsState['allIds']>([], builder =>
+    builder
+        .addCase(setAnnotation, (state, { payload: { id } }) => {
+            if (!state.includes(id)) state.push(id);
```

Comment:
Shouldn't the lack of curly braces result in an eslint error?

Reply:
Apparently not. I don't mind this syntax and there's no lint rule against it. Thoughts?

Reply:
I guess it's a recommended rule for a reason? Mainly readability and debuggability probably

Reply:
Fine, fine. 😄